backend/tests/test_capabilities.py

```
@@ -0,0 +1,243 @@
+            identity.peer_id,
+            identity.signing_public_key_bytes(),
+            identity.encryption_public_key_bytes(),
+            identity.display_name,
+            b"2" * 32,
+            b"",
+            b"",
+            [CAP_TEXT_CHAT],
+        )
+        payload.signature = identity.signing_private_key.sign(payload.signed_bytes())
+        payload.capabilities.append(FUTURE_CAPABILITY)
+        with self.assertRaises(Exception):
```

Ruff (0.16.2)

[warning] Do not assert blind exception: `Exception`

(B017)